components/scrapd-home-view/component.js

```diff
@@ -5,19 +5,24 @@ const ScrapdHomeView = () => {
   const intro = [
     <div key="0">
       <p>
-        ScrAPD is a collection of tools to automatically collect information about traffic fatalities in the city of
-        Austin, TX and render it on a modern dashboard.
+        ScrAPD is a web scraper that continuously and automatically collects traffic fatality information for the city
```

Comment:
ScrAPD is not simply a scraper, it is a collection of 4 tools: the scraper, the pipelines, the data sets, and the dashboard.

Reply:
Is it inappropriate do you know to use `web scraper` in the context of this sentence, in the section on Data sets?

>The ScrAPD web scraper generates data sets which populate our ScrAPD dashboard.

Reply:
Complete!